OrbitGl/App.cpp

```diff
@@ -40,7 +40,11 @@
 #include "StringManager.h"
 #include "Utils.h"
 
+ABSL_FLAG(bool, enable_tracepoint_service, false,
+          "Start the retrieval of kernel tracepoints from the service");
+
 ABSL_DECLARE_FLAG(bool, devmode);
+ABSL_DECLARE_FLAG(bool, enable_tracepoint_service);
```

Comment:
You only need ABSL_DECLARE_FLAG if you want to access the flag in a different module than the one you defined it in. Since you  have the ABSL_FLAG in this file you can just remove ABSL_DECLARE_FLAG here. 

That said, can you get away with just one flag? Having enable_tracepoint_service and enable_tracepoint_feature seems redundant.

https://abseil.io/docs/cpp/guides/flags

Reply:
If I get it right you should be able to put a single ABSL_FLAG into App.cpp and a ABSL_DECLARE_FLAG into App.h. Since OrbitQt\orbitmainwindow.cpp includes App.h it will be visible there.